libmproxy/models/http.py

```diff
@@ -11,9 +11,10 @@
 from .. import version, stateobject
 from .flow import Flow
 
+from collections import OrderedDict
 
 class MessageMixin(stateobject.StateObject):
-    _stateobject_attributes = dict(
+    _stateobject_attributes = OrderedDict(
```

Comment:
Yes, you are correct. 
I checked what `dict.iteritems()` returns on python2.7 and pypy. And the order in which the items are yielded is different. This results in a data dependency between `headers` and `content`, since setting the content also adds a header, which might be overwritten later.